docker-bake.hcl

```diff
@@ -27,6 +27,21 @@ variable "TIMESTAMP" {
   default = ""
 }
 
+# Group of Deno targets
+group "deno" {
+  targets = [
+    "deno_1_37-dev",
+  ]
+}
+
+# Group of Deno 1.37 targets
+group "deno_1_37-dev" {
+  targets = [
+    "deno_base_1_37-dev",
```

Comment:
This target would need to also be added to this file. something like 
```hcl 
target "deno_base_1_37-dev" {
  inherits = ["base"]
  context = "deno/base/1.37-dev"
  tags = [
    "ghcr.io/noteable-io/kernel-deno-1.37-base:${TAG}",
    notequal("", HANDLE) ? "ghcr.io/noteable-io/kernel-deno-1.37-base:${HANDLE}" : ""
  ]
}
```
Just for edification:
- `target` references an object that builds an actual artifact
- `group` is simply a helper to execute multiple targets at once

Reply:
Thanks. I clearly didn't copy the rust kernel images enough. 😆